_drafts/mastering-with-git.markdown

```diff
@@ -0,0 +1,86 @@
+---
+layout: post
+title:  "Mastering git with team"
+date:   2016-12-6 12:55:28 +0100
+categories: devops project management versioning workflow
+---
+## Introduction
+Work on project with people must be controlled by rules, systems, and best
+pratices work ensure the long life of the project.
+
+Working on project can be addressed to different publics like, a complet project team (private or free projects), 
+simple user who want to contribute to an upstream project, or also core team of a project who want work with external contributors.
+
+This post introduce a complete describe of these use cases and offer to you step by step explaination of these and the 
+best pratices for manage them properly.
+
+First, we introduce a global overview of what was a project? 
+This part can interess people who want to setup a complete workflow on project management.
+
+Seconde, we explain to users how to contribute to projects. Considere users on this part
+who was external to the core team of a product and doesn't have the right to write directly
+on the upstream project.
+
+Third, we will help core team to manage workflow and work with contributors.
+
+if you are familiar with these concept you doesn't have read this post entirely and you can switch 
+directly in the part of your choice.
+
+Folks who manage projects like DevOps, project manager, project owner, etc can be interested by the 
```

Comment:
I'd used People instead of Folks ... but I'd rather 
Project managers, DevOps engineers, projects owners ... should be interested by all  parts of this post.